Structured Random Phase Retrieval (#332)
deepinv/deepinv: docs/source/deepinv.denoisers.rst

Complex Denoisers
--------------------------

> Originally, the denoisers are designed to process real-valued images. However, some problems, e.g., phase retrieval, require processing complex-valued images. We hence provide a function :class:`deepinv.models.complex.to_complex_denoiser` which can convert any real-valued denoiser into a complex-valued denoiser. It can be simply called by ``complex_denoiser = to_complex_denoiser(denoiser)``.

Reviewer: ```suggestion
Most denoisers are designed to process real-valued images. However, some problems, e.g., phase retrieval, require processing complex-valued images. We hence provide a function :class:`deepinv.models.complex.to_complex_denoiser` which can convert any real-valued denoiser into a complex-valued denoiser. It can be simply called by ``complex_denoiser = to_complex_denoiser(denoiser)``.
```
